_posts/2020-02-16-walking.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Walking Is A Choice
+date: 2020-02-15 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+## Walking
+
+Walking is a choice.
+
+I have an office job and if I wanted to I could just sit all day in my chair without moving at all (assuming I do not have meetings that day). For some time I spent the time at my job like that, like I expect most people do as well.
+
+Starting last year I decided to take a glass of (still) water and put it on my desk. A small glass. Every single day.
+
+Each time I take a pause I see the glass and take a sip. When the water is finished I walk to the water dispenser furthest away from my desk to refill it. The office is not huge so I do not waste more than a minute doing it, but I also walk a bit. I do this on average 5 times a day. An added bonus is that I also rest my eyes from the screen for this short time[^1].
+
+On top of that I try to spend at least one hour on the [treadmill][1] every day I'm in the office, but I realize that most offices do not have it so that's not usually an option.
+
```

Comment:
You already talked about the water glass before... so add some more stuff here or smth

Reply:
Really? lol 

Nothing for now, will leave it then for now and may or may not come back to this later

Reply:
I think this counts as a bit more..?

Reply:
@MsSlacker